build

```diff
@@ -3,14 +3,29 @@
 ORG_PATH="github.com/coreos"
 REPO_PATH="${ORG_PATH}/etcd"
 
+GIT_SHA=`git rev-parse --short HEAD || echo "GitNotFound"`
+
+LINK_OPERATOR="="
+
+: ${ARCH_LIST:="amd64 arm64"}
```

Comment:
@glevand I think CI would pick it up with targeted builds anyway. I don't think anyone is actually patient enough to regularly run the full test suite before submitting a PR. I'd prefer to hold off on adding an extra option to ./build until it becomes obviously necessary...